Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -130,6 +131,9 @@ @implementation WKFullScreenViewController {
 #if ENABLE(FULLSCREEN_DISMISSAL_GESTURES)
     RetainPtr<UIStackView> _banner;
     RetainPtr<_WKInsetLabel> _bannerLabel;
+    RetainPtr<UITapGestureRecognizer> _bannerTapToDismissRecognizer;
+    BOOL _blockHideBanner;
+    BOOL _hideBannerAfterMinimumHideDelayReached;
```

Comment:
I feel the logic would be easier to follow if we simply stored a timestamp of last banner presentation.

If the difference is less than `minimumHideDelay`, you would call `performSelector:afterDelay` with the difference.